test/Microsoft.Dnx.ApplicationHost.FunctionalTests/AppHostTests.cs

```diff
@@ -20,10 +20,9 @@ public void LibraryExporterGetExports(DnxSdk sdk)
             sdk.Dnu.Restore(project.ProjectDirectory).EnsureSuccess();
 
             // Act
-            var result = sdk.Dnx.Execute(project);
+            var result = sdk.Dnx.Execute(project).EnsureSuccess();
```

Comment:
Do the other tests need in this type need to be changed to do something similar?